scripts/deploy.sh

```diff
@@ -114,6 +114,17 @@ if [[ "${DEPLOY_OA}" == "yes" ]]; then
   # setup the infrastructure
   run_ansible setup-infrastructure.yml
 
+  # When running in an AIO, we need to drop the following iptables rule in any neutron_agent containers
+  # to that ensure instances can communicate with the neutron metadata service.
```

Comment:
really minor comment - but "to ensure that" 
Since it hasnt merged yet may as well just fix that.

Otherwise LGTM